### PR TITLE
[1.2] rbac: add tests for path with double dots (#14184)

### PR DESCRIPTION
**Pull request by yangminzhu**
Cherrypick https://github.com/istio/istio/pull/14184 from master.

### CODE REVIEW

File: tests/integration/security/rbac/v1/path/main_test.go [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+	p    pilot.Instance
+)
+
+func setupConfig(cfg *istio.Config) {
+	if cfg == nil {
+		return
+	}
+	cfg.Values["sidecarInjectorWebhook.rewriteAppHTTPProbe"] = "true"
+
+	// TODO(https://github.com/istio/istio/issues/14084) remove this
```
Comment (fpesce) [inline tests/integration/security/rbac/v1/path/main_test.go]: #14084 is closed, what is the point of this comment?

Comment (yangminzhu) [reply tests/integration/security/rbac/v1/path/main_test.go]: I think @howardjohn just send out the PR to clean this in master: https://github.com/istio/istio/pull/14602

**Comment by yangminzhu** [comment]
@fpesce the tests are passed, can we merge? Thanks.

**Pull request merged**